nam/models/base.py

```diff
@@ -173,6 +175,15 @@ def parse_config(cls, config):
     def net(self) -> nn.Module:
         return self._net
 
+    def initialize_losses(self):
+        if self._loss_config.mstft_weight > 0.0:
+            import auraloss
+            self._mrstft = auraloss.freq.MultiResolutionSTFTLoss()
```

Comment:
What's the reason for doing this instead of 
```python
from .losses import mrstft_loss

# then e.g. line 219
loss = loss + self._loss_config.mstft_weight * mrstft_loss(preds, targets)
```
?

Seems like we could cut out lines 178-186 and not need to define `self._mrstft`.

If there's not a clear advantage, then I'd prefer what I've written here so that it's consistent with the other losses' implementations.

Reply:
Sure, but we need to instantiate the MultiResolutionSTFTLoss and store it somewhere - I think instantiating it in every loss function call is not a good idea. All the previous losses are stateless and therefore this needs different handling. https://github.com/csteinmetz1/auraloss#usage

Another alternative would be to have AuraLoss class defined in losses.py and its instance would be stored in Model.